charts/timed/templates/job-dbmigrate.yaml

```diff
@@ -3,14 +3,14 @@ kind: Job
 metadata:
   # TODO: this should be done during pre phase but to do so following
   # feature is needed: https://github.com/kubernetes/helm/issues/2243
-  name: post-upgrade-{{ template "timed.fullname" . }}-dbmigrate
+  name: {{ template "timed.fullname" . }}-dbmigrate
   labels:
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
     chart: {{ .Chart.Name }}-{{ .Chart.Version }}
     app: {{ template "timed.name" . }}
   annotations:
-    "helm.sh/hook": post-upgrade
+    "helm.sh/hook": post-install,post-upgrade
```

Comment:
Do you need to change this in `charts/timed/templates/pre-install-secret-backend.yaml` as well?

Reply:
no because we do not want the secret (which just contains DJANGO_SECRET_KEY) to be regenerated on upgrades. But we want migrations to be run on upgrades.